code/iaas/logic-common/src/main/java/io/cattle/platform/process/common/lock/MountVolumeLock.java

```diff
@@ -0,0 +1,17 @@
+package io.cattle.platform.process.common.lock;
+
+import io.cattle.platform.lock.definition.AbstractLockDefinition;
+import io.cattle.platform.lock.definition.BlockingLockDefinition;
+
+public class MountVolumeLock extends AbstractLockDefinition implements BlockingLockDefinition {
```

Comment:
You can extend AbstractBlockingLockDefintion and then getWait() isn't required and will default to `default.lock.wait.millis`